hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -268,7 +286,7 @@ public String getCACertificate() throws IOException {
   public List<String> listCertificate(NodeType role,
       long startSerialId, int count, boolean isRevoked) throws IOException {
     List<X509Certificate> certificates =
-        certificateServer.listCertificate(role, startSerialId, count,
+        rootCertificateServer.listCertificate(role, startSerialId, count,
```

Comment:
This API semantics is not changed it can return list of certificates for all roles.
As scmcertserver runs on all SCM's used the scmcertserver

Underlying both CertificateServers use the same certstore. So, the semantics of this API is not changed, it can get any certificate.